components/TextFields/tests/unit/TextFieldTests.swift

```diff
@@ -100,10 +129,29 @@ class TextFieldTests: XCTestCase {
 
     XCTAssertTrue(textField.subviews.contains(leftView))
     XCTAssertTrue(textField.subviews.contains(rightView))
+
+    XCTAssertEqual(textField.leadingView, leftView)
```

Comment:
I would LOVE to have actual RTL specific test code but we don't right now. I could add it in but I'd add a bunch more if I knew we had it.